docs/cub/developer_overview.rst

```diff
@@ -594,6 +594,14 @@ All the kernel does is derive the proper policy type,
 unwrap the policy to initialize the agent and call one of its ``Consume`` / ``Process`` functions.
 Agents are frequently reused by unrelated device-scope algorithms.
 
+.. _cub-developer-policies:
+
+Policies
+====================================
+
+Policies describe the configuration of agents wrt. to their execution.
```

Comment:
Does this document define "agent" anywhere?

Reply:
Yes, once sentence above it says:

> Agents are frequently reused by unrelated device-scope algorithms.

I extended it to:

"Agents hold kernel bodies and are frequently reused by unrelated device-scope algorithms."